### ISSUE TITLE
Fullscreen subtitles are at the top instead of the bottom

### ISSUE THREAD

**Issue by jhon1282** [revanced/revanced-patches#1750]
### Type

Cosmetic

### Bug description

Fullscreen subtitles are at the top instead of the bottom

### Steps to reproduce

Start video in full screen
Select subtitles 

### Relevant log output

```shell
_
```

### Screenshots or videos

![Screenshot_2023-03-14-22-18-52-197_app revanced android youtubf](https://user-images.githubusercontent.com/125223735/225139834-37d7a547-2cb1-413b-a8d5-c963fdbfd16b.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Side effect of the recent patch for video playback

**Comment by sha-265** [comment]
Confirmed

**Comment by SodaWithoutSparkles** [comment]
Any possible fix?

**Comment by SodaWithoutSparkles** [comment]
To those who stumbled upon this: subtitles at the top blocks many contents. Changing the background visibility (the black box) to somthing like 50% or 25% would help a lot. You just need to go to revanced > settings (not revanced spectific settings) > subtitles > subtitle size and style > subtitle style and change it to customize. In the new menu appeared, scroll down and find background visibility, change it to 50% or 25%. 

This would allow you to still see some content behind the subtitle, instead of a solid black. You could tweak the settings further to suit your likes, but do not touch the subtitle window colour settings, it would create a solid coloured bar right across the entire screen.

**Comment by oSumAtrIX** [comment]
Closing in favour of #1752

**Comment by trmdi** [comment]
> Side effect of the recent patch for video playback

Is this really caused by the patch? Because on the unpatched Youtube, the CC also shows up on the top with some videos (but not all)

**Comment by SodaWithoutSparkles** [comment]
Youtubers can configure the CC subtitle position.

**Comment by jaehyuneybuns** [comment]
> Youtubers can configure the CC subtitle position.

But that's not the case here. It's for every channel. Even the ones that were at the bottom prior to using the patch.

**Comment by jaehyuneybuns** [comment]
> > Side effect of the recent patch for video playback
> 
> Is this really caused by the patch? Because on the unpatched Youtube, the CC also shows up on the top with some videos (but not all)

Yes. After I started using the patch, all subtitles moved to the top for me. Even on channels where it was always at the bottom. I've never had this happen before.

**Comment by sha-265** [comment]
This issue should be reopened probably.

**Comment by oSumAtrIX** [comment]
#1752

**Comment by SodaWithoutSparkles** [comment]
@jaehyuneybuns @sha-265 checkout #2003
